_posts/2024-05-14-pyos-guide-to-pyconus-2024.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: single
+title: "The pyOpenSci Guide to PyCon 2024"
+excerpt: "PyCon US 2024 is finally here, and we can't wait to connect with you! This post has all of the talks, panels, summits, and keynotes where pyOpenSci community members (and friends!) will be speaking."
+author: "Jesse Mostipak"
+permalink: /blog/pyos-guide-to-pyconus-2024.html
+header:
+  overlay_image: images/blog/2024/may/pycon-us-2024.png
+  overlay_filter: 0.6
+categories:
+  - blog-post
+  - community
+classes: wide
+toc: true
+comments: true
+---
+## <i class="fa-solid fa-heart"></i> Connect with the pyOpenSci community at PyCon US 2024!
+We know there are a million things to do and see while at [PyCon US 2024](https://us.pycon.org/2024/) in beautiful Pittsburgh this weekend, so we wanted to compile a list of every talk, keynote, summit, and panel where you can expect to meet some of the fantastic pyOpenSci community members!
```

Comment:
one last comment. 
just looking at this language, can we relax it a bit? i'm not sure how to do this.

For instance sumana - she is giving a keynote and it an incredible person who i want to highlight. i don't know her well so she might be surprised to be considered "part of pyos community" ... similarly sarah kaiser might also be surprised? i may be WAY too sensitive i just don't want some of the people here who i really respect but haven't been directly engaged with pyos (vs i have spoken with them individually, etc) to be surprised by the language. Maybe it could say somethin glike - every talk keynote ... where ou can meet pyos community members and other people doing incredibly important work in the python community (or whatever words you think is best just to relax things a bit).

Reply:
not a problem!